scaleway/table_scaleway_account_project.go

```diff
@@ -0,0 +1,133 @@
+package scaleway
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/scaleway/scaleway-sdk-go/api/account/v3"
+	"github.com/scaleway/scaleway-sdk-go/scw"
+
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableScalewayAccountProject(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "scaleway_account_project",
+		Description: "A Scaleway Account Project.",
+		List: &plugin.ListConfig{
+			Hydrate: listAccountProjects,
+		},
+		Columns: []*plugin.Column{
+			{
+				Name:        "project_id",
+				Description: "The ID of the project.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("ID"),
+			},
+			{
+				Name:        "name",
+				Description: "Name of the project.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "created_at",
+				Description: "The time when the project was created.",
+				Type:        proto.ColumnType_TIMESTAMP,
+			},
+			{
+				Name:        "updated_at",
+				Description: "The time when the project was updated.",
+				Type:        proto.ColumnType_TIMESTAMP,
+			},
+			{
+				Name:        "description",
+				Description: "Description of the project.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "organization",
+				Description: "Organization ID of the project.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("OrganizationID"),
+			},
+		},
+	}
+}
+
+//// LIST FUNCTION
+
+func listAccountProjects(ctx context.Context, d *plugin.QueryData, _ *plugin.HydrateData) (interface{}, error) {
+
+	// Create client
+	client, err := getSessionConfig(ctx, d)
+	if err != nil {
+		plugin.Logger(ctx).Error("scaleway_project.listProjects", "connection_error", err)
+		return nil, err
+	}
+
+	// Create SDK objects for Scaleway Account product
+	accountApi := account.NewProjectAPI(client)
+
+	// Get organisationID from config to request IAM API
+	organisationId := GetConfig(d.Connection).OrganizationID
+	if organisationId == nil {
+		err := fmt.Errorf("missing organization_id in scaleway.spc")
+		plugin.Logger(ctx).Error("scaleway_project.listProjects", "query_error", err)
+		return nil, err
+	}
+
+	quals := d.EqualsQuals
+
+	req := &account.ProjectAPIListProjectsRequest{
+		OrganizationID: *organisationId,
+		Page:           scw.Int32Ptr(1),
+	}
+	// Additional filters
+	if quals["name"] != nil {
```

Comment:
If we can pass the `name` column in, can you please add the `name` column as an optional key column to the `ListConfig` `KeyColumns`? This helps Steampipe cache results correctly based on what users pass into the qualifiers.

Reply:
i have had KeyColumns.